src/libraries/System.Net.Ping/src/System/Net/NetworkInformation/Ping.PingUtility.cs

```diff
@@ -15,6 +15,8 @@ namespace System.Net.NetworkInformation
 {
     public partial class Ping
     {
+        private static char[] s_ttlExceededAddressSeparators = new[] { ' ', ':' };
```

Comment:
How would this work for IPv6?

Reply:
If I understand it correctly, we only use this for the IPv4 ping, not for the IPv6. The v6 output format is `From 2a01:4f8:0:e172::1801 icmp_seq=1 Time exceeded: Hop limit` and so the check on line 131/133 will exit early.

Reply:
ah. I wish we have explicit `AddressFamily` check. I'm also wondering if we miss IPv6 in #65312 @rzikm? It seems like the test you were trying to fix may in practice do inly IPv4. We _should_ be able to set Hop limit and report intermediate hops.